Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
```

Comment:
this is a pretty chunky image, `python:3-slim` or `python:3-alpine` (though, generally I don't suggest using alpine) are probably better choices

```console
$ docker images | grep python.*3
docker.io/library/python                        3-slim    6cf621cb1327  4 days ago     118 MB
docker.io/library/python                        3         28a4c88cdbbf  4 days ago     905 MB
docker.io/library/python                        3-alpine  0f03316d4a27  6 days ago     44.7 MB
```